style/style.css

```diff
@@ -0,0 +1,668 @@
+    height: 82.06px;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 10.7034px;
+    line-height: 22px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.06em;
+    color: #fff;
+  }
+}
+
+.footer-desktop-img {
+  width: 325px;
+  height: 61px;
+  margin-top: 50px;
+  margin-bottom: 30px;
+  margin-left: 100px;
+}
+
+.footer-desktop-p {
+  width: 338px;
+  height: 17px;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 28px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.01em;
+  margin-top: 60px;
+  margin-bottom: 30px;
+  margin-left: 100px;
+  color: #000;
+}
+
+.card-img-top {
+  width: 178px;
+  height: 100%;
+}
+
+.card-img-no-padding {
+  margin: 0;
+  padding: 0;
+  text-align: left !important;
+}
+
+.speakers .card {
+  align-items: flex-start !important;
+  text-align: left !important;
+  width: 600px;
+  height: 177px;
+  border: none;
+}
+
+.card-text-two {
+  color: #f78c89;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 15px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.02em;
+  margin-top: 15px;
+}
+
+.card-text-three {
+  color: #000;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 15px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.02em;
+  margin-top: 15px;
+}
+
+.text-heading {
+  margin-bottom: 55px;
+}
+
+@media (max-width: 768px) {
+  .card-img-top {
+    width: 107px;
+    height: 106.4px;
+  }
+
+  .card-img-no-padding {
+    margin-left: -7px;
+    padding: 0;
+    text-align: left !important;
+  }
+
+  .speakers .card {
+    align-items: flex-start !important;
+    text-align: left !important;
+    width: 310px;
+    height: 162px;
+  }
+
+  .card-text-two {
+    color: #f78c89;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 10px;
+    line-height: 12px;
+    display: flex;
+    align-items: flex-start;
+    letter-spacing: -0.02em;
+    margin-top: 10px;
+    margin-left: -42px;
+    width: 200px;
+  }
+
+  .card-text-three {
+    color: #000;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 10px;
+    line-height: 12px;
+    display: flex;
+    align-items: flex-start;
+    letter-spacing: -0.02em;
+    margin-top: 10px;
+    margin-left: -42px;
+    width: 200px;
+  }
+
+  .card-title-h {
+    height: 24.08px;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 14.2712px;
+    line-height: 25px;
+    display: flex;
+    align-items: center;
+    letter-spacing: -0.01em;
+    color: #000;
+    margin-left: -42px;
+    width: 174px;
+  }
+}
+
+@media (max-width: 767px) { /* Apply styles on devices with a width of 767px or less (mobile) */
```

Comment:
- [ ] Hi, I noticed that your CSS styling is not built as mobile-first. You are using `@media (max-width:)` to adjust your layout for smaller screens, but this is not the best practice. Mobile-first design means that you start with the smallest screen size and then use `@media (min-width:)` to add more styles for larger screens. This will make your code more efficient and responsive, and also improve your performance and user experience. I suggest you refactor your CSS code to follow the mobile-first approach and see how it improves your design. You can read more about it here: https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Responsive/Mobile_first. Let me know if you have any questions! 😊